subsys/bluetooth/common/Makefile

```diff
@@ -0,0 +1,3 @@
+ccflags-y +=-I$(srctree)/subsys/bluetooth
+obj-$(CONFIG_BLUETOOTH_DEBUG) += log.o
```

Comment:
Because it only really deals with logging. If anything I would change CONFIG_BLUETOOTH_DEBUG to CONFIG_BLUETOOTH_LOGGING